oaiharvest/harvest.py

```diff
@@ -207,11 +240,6 @@ def main(argv=None):
     else:
         args = argparser.parse_args(argv)
     logger = logging.getLogger(__name__).getChild('main')
-    # Parse from and until into datetime objects
```

Comment:
Did you mean to remove the functionality to selectively harvest records updated between specified dates?